examples/kubernetes/vttablet-pod-template.yaml

```diff
@@ -57,6 +57,7 @@ desiredState:
             -db-config-filtered-uname vt_filtered
             -db-config-filtered-dbname vt_{{keyspace}}
             -db-config-filtered-charset utf8
+            -queryserver-config-transaction-cap 300
```

Comment:
I assume this was pulled from the 'benchmarking' branch. I don't think we should alter the default example templates here. If we really need to, we could create a new template specifically for benchmarking.